scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -62,14 +71,28 @@ fi
 bundle_id=${1:-$HOSTNAME}
 dest_dir=${2:-$DEFAULT_DEST_DIR}
 
-while getopts ":b:t:c:s:" flag
-do
-    case "${flag}" in
-          b) bundle_id="$OPTARG";;
-          t) dest_dir="$OPTARG";;
-          c) config_url="$OPTARG";;
-          s) sevice="$OPTARG";;
-    esac
+duration=P5d
+size_limit=0
+services=All
+stacktrace=false
+coredumps=false
+binlogs=true
+TEMP=$(getopt -o c: --long duration:,size_limit:,services:,binlogs:,coredumps:,stacktrace: \
```

Comment:
As you have initialized these values to (true/false), it seems Boolean options. Don't need to use colon (:)  for these options, this can be make no-args options.
It depends on how caller is going to call this, if it is like "--binlogs true/false" then  this is fine.

Reply:
Example of support bundle generation command line:
/opt/seagate/cortx/motr/libexec/m0reportbug-bundler SBbw5f7n9i /var/log/support_bundle --duration "2020-09-06T05:30:00P5DT3H3S" \
--size_limit 100MB --services All --binlogs false --coredumps false --stacktrace false